extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/mongodb/LiquibaseMongodbFactory.java

```diff
@@ -51,6 +51,8 @@ private ResourceAccessor resolveResourceAccessor() throws FileNotFoundException
         }
 
         CompositeResourceAccessor compositeResourceAccessor = new CompositeResourceAccessor();
+        compositeResourceAccessor
+                .addResourceAccessor(new ClassLoaderResourceAccessor(Thread.currentThread().getContextClassLoader()));
```

Comment:
Why this change?

Reply:
It was something I realized along the way that it can be useful to have a changelog within the classpath that references files on and off the classpath as shown below.

![image](https://github.com/quarkusio/quarkus/assets/51928477/e079081a-b0e1-45cd-b735-defec653c638)

Reply:
Thanks for the information.

Reply:
In this case I think this change should be a separate commit as it's pretty much unrelated to the rest of the changes in PR

Reply:
Understood, but then I put it in a different commit or in a different pr?